tests/test_flows.py

```diff
@@ -633,11 +633,23 @@ def test_link_metric_query(self):
 
     def test_combined_infra_metrics(self):
 
-        self.send_and_check_policy_config_metric_reporting(env.agents[0], True, False)
-
-        debug("Send AP Metrics query message to agent 1")
+        sta1 = env.Station.create()
+        sta2 = env.Station.create()
         vap1 = env.agents[0].radios[0].vaps[0]
         vap2 = env.agents[1].radios[1].vaps[0]
+        vap1.associate(sta1)
+        vap2.associate(sta2)
+
+        # Set station link metrics
+        # TODO make abstraction for this in Radio
+        env.agents[1].radios[1].send_bwl_event(
+            "DATA STA-UPDATE-STATS {} rssi=-38,-39,-40,-41 snr=38,39,40,41 "
+            "uplink=1000 downlink=800".format(sta2.mac))
+
+        time.sleep(1)
+
+        debug("Send AP Metrics query message to agent 1 with Traffic Stats for {}".format(sta1.mac))
```

Comment:
Debug message is mixing AP Metrics Query message with Multi-AP Policy Config Request message

Reply:
Fixed